src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -38,11 +38,11 @@ public final void init() {
 
   //Set open and closed loop ramp rates
   shoulderFalcon.configOpenloopRamp(0);
-  shoulderFalcon.configClosedloopRamp(0);
```

Comment:
Some comments in here explaining why the change from 0 to 0.5 would be good.  It could help the next person to know what was used before and what the difference in behavior was like.